content/source/docs/enterprise/api/teams.html.md

```diff
@@ -75,8 +131,59 @@ $ curl \
 }
 ```
 
+## Show Team Information
 
-# Delete a Team
+`GET /teams/:team_id`
```

Comment:
This is more a comment on the code, I guess, but most of our "IDs" in the API are external IDs, it would be probably better if teams worked the same way, since they do seem to have external IDs (per the above additions).